src/disco/constants.js

```diff
@@ -0,0 +1,6 @@
+export const DOWNLOADING = 'downloading';
```

Comment:
Ok - I think we should see when we need that - we might share most of this and it will just work. Happy to fix it up if it does become unwieldy though.